### ISSUE TITLE
PingPong CAS latency test seems to broken ... 

### ISSUE THREAD

**Issue by Cotton-Ben** [OpenHFT/Chronicle-Map#216]
Hi Chronicle.

I am having trouble with the old JPMorgan test using ChronicleMap as off-heap operand for (bond coupon mutations) exercising latency of CAS operators (across 2 JVM processes).

https://github.com/OpenHFT/Chronicle-Map/blob/master/src/test/java/net/openhft/chronicle/map/fromdocs/pingpong_latency/PingPongCASLeft.java

Running this Test on Windows10 laptop (not Linux /dev/shm!), using Java 8, I get the attached Exception at this piece of code:

line 56
```
 BondVOInterface bond1 = newNativeReference(BondVOInterface.class);

```
Which, as I recall from the old InfoQ paper, is where Chronicle uses an in-process 'javac' capability to render an off-heap 'bond' impl from the user defined interface.

If any one could effort providing remedy so that we do not endure the attached Exception (on Windows), then know that we are grateful: this test is a *central* masterpiece when demonstrating Chronicle.
```
attached:

[main] INFO net.openhft.affinity.Affinity - Set main to thread id 8140
Exception in thread "main" java.lang.RuntimeException: java.lang.NoClassDefFoundError: com/sun/source/util/Trees
	at com.sun.tools.javac.main.Main.compile(Main.java:553)
	at com.sun.tools.javac.api.JavacTaskImpl.doCall(JavacTaskImpl.java:129)
	at com.sun.tools.javac.api.JavacTaskImpl.call(JavacTaskImpl.java:138)
	at net.openhft.chronicle.values.CachedCompiler.compileFromJava(CachedCompiler.java:62)
	at net.openhft.chronicle.values.CachedCompiler.loadFromJava(CachedCompiler.java:86)
	at net.openhft.chronicle.values.ValueModel.createClass(ValueModel.java:323)
	at net.openhft.chronicle.values.ValueModel.createNativeClass(ValueModel.java:303)
	at net.openhft.chronicle.values.ValueModel.nativeClass(ValueModel.java:279)
	at net.openhft.chronicle.values.Values.nativeClassFor(Values.java:86)
	at net.openhft.chronicle.values.ValueFieldModel$NativeMemberGenerator.generateFields(ValueFieldModel.java:207)
	at net.openhft.chronicle.values.ValueFieldModel$NativeMemberGenerator.generateArrayElementFields(ValueFieldModel.java:218)
	at net.openhft.chronicle.values.ArrayFieldModel$ArrayMemberGenerator.generateFields(ArrayFieldModel.java:152)
	at net.openhft.chronicle.values.FieldModel.generateMembers(FieldModel.java:200)
	at net.openhft.chronicle.values.FieldModel.generateNativeMembers(FieldModel.java:192)
	at net.openhft.chronicle.values.Generators.lambda$generateNativeClass$0(Generators.java:42)
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1382)
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:580)
	at net.openhft.chronicle.values.Generators.generateNativeClass(Generators.java:42)
	at net.openhft.chronicle.values.ValueModel.createClass(ValueModel.java:321)
	at net.openhft.chronicle.values.ValueModel.createNativeClass(ValueModel.java:303)
	at net.openhft.chronicle.values.ValueModel.nativeClass(ValueModel.java:279)
	at net.openhft.chronicle.values.Values.nativeClassFor(Values.java:86)
	at net.openhft.chronicle.values.Values.newNativeReference(Values.java:54)
	at net.openhft.chronicle.map.fromdocs.pingpong_latency.PingPongCASLeft.playPingPong(PingPongCASLeft.java:56)
	at net.openhft.chronicle.map.fromdocs.pingpong_latency.PingPongCASLeft.main(PingPongCASLeft.java:51)
Caused by: java.lang.NoClassDefFoundError: com/sun/source/util/Trees
	at net.openhft.sg.ContextProcessor.init(ContextProcessor.java:51)
	at com.sun.tools.javac.processing.JavacProcessingEnvironment$ProcessorState.<init>(JavacProcessingEnvironment.java:500)
	at com.sun.tools.javac.processing.JavacProcessingEnvironment$DiscoveredProcessors$ProcessorStateIterator.next(JavacProcessingEnvironment.java:597)
	at com.sun.tools.javac.processing.JavacProcessingEnvironment.discoverAndRunProcs(JavacProcessingEnvironment.java:690)
	at com.sun.tools.javac.processing.JavacProcessingEnvironment.access$1800(JavacProcessingEnvironment.java:91)
	at com.sun.tools.javac.processing.JavacProcessingEnvironment$Round.run(JavacProcessingEnvironment.java:1035)
	at com.sun.tools.javac.processing.JavacProcessingEnvironment.doProcessing(JavacProcessingEnvironment.java:1176)
	at com.sun.tools.javac.main.JavaCompiler.processAnnotations(JavaCompiler.java:1170)
	at com.sun.tools.javac.main.JavaCompiler.compile(JavaCompiler.java:856)
	at com.sun.tools.javac.main.Main.compile(Main.java:523)
	... 24 more

Process finished with exit code 1

```

**Comment by Cotton-Ben** [comment]
Disregard!  Now working.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Map-3.19.30](https://github.com/OpenHFT/Chronicle-Map/releases/chronicle-map-3.19.30), [BOM-2.19.176](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.19.176)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Map-3.19.30](https://github.com/OpenHFT/Chronicle-Map/releases/chronicle-map-3.19.30), [BOM-2.19.179](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.19.179)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Map-3.19.30](https://github.com/OpenHFT/Chronicle-Map/releases/chronicle-map-3.19.30), [BOM-2.19.180](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.19.180)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Map-3.19.30](https://github.com/OpenHFT/Chronicle-Map/releases/chronicle-map-3.19.30), [BOM-2.19.181](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.19.181)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Map-3.19.30](https://github.com/OpenHFT/Chronicle-Map/releases/chronicle-map-3.19.30), [BOM-2.19.182](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.19.182)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Map-3.19.30](https://github.com/OpenHFT/Chronicle-Map/releases/chronicle-map-3.19.30), [BOM-2.19.184](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.19.184)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Map-3.19.30](https://github.com/OpenHFT/Chronicle-Map/releases/chronicle-map-3.19.30), [BOM-2.19.185](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.19.185)
